pages/api/products/list.ts

```diff
@@ -5,13 +5,12 @@ export default async function list(req: NextApiRequest, res: NextApiResponse) {
     try {
         const { accessToken, storeHash } = await getSession(req);
         const bigcommerce = bigcommerceClient(accessToken, storeHash);
-        // Optional: pass in API params here
-        const params = [
-            'limit=11',
-        ].join('&');
+        const { page, limit, sort, direction } = req.query;
 
-        const { data } = await bigcommerce.get(`/catalog/products?${params}`);
-        res.status(200).json(data);
+        const response = await bigcommerce.get(
+          `/catalog/products?page=${page}&limit=${limit}&sort=${sort}&direction=${direction}`
```

Comment:
since the params have expanded, we should probably make use of `URLSearchParams` - even more so given that sort and direction should be optional

Reply:
e.g.
```jsx
const { page, limit, sort, direction } = req.query;
const params = new URLSearchParams({ page, limit, ...(sort && { sort, direction }) }).toString();

const response = await bigcommerce.get(`/catalog/products?${params}`);
```